OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/util/ApplicationUtils.java

```diff
@@ -74,6 +81,31 @@ public static String getAppDirectory() {
         return tempDir.getAbsolutePath();
     }
     
+    public static String getAppVersion() {
+        URL url = ApplicationUtils.class.getClassLoader().getResource(APP_VERSION_FILENAME);
```

Comment:
We shouldn't do all of this ourselves. You can use class Properties, that does exactly the same, in one line "load(InputStream is)"